integrations/tanstack-router/index.html

```
@@ -0,0 +1,2 @@
+<div id="root"></div>
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)